src/blueapi/config.py

```diff
@@ -21,13 +21,23 @@ class Source(BaseModel):
     module: Union[Path, str]
 
 
+class BasicAuthentication(BaseModel):
```

Comment:
I think username/password is generic enough that it might end up being useful elsewhere (e.g. it's exactly the same for communicating with RabbitMQ over AMQP), and having to change it back later is more of a nuisance than accepting that it's a general thing.

Happy to change if you're holding firm on it, but it was StompAuthentication until I went back to work on some AMQP things and changed it almost immediately.